jerry-ext/arg/arg-transform-functions.c

```diff
@@ -132,7 +132,7 @@ jerryx_arg_helper_process_double (double *d, /**< [in, out] the number to be pro
                                   double max, /**< the max value for clamping */
                                   jerryx_arg_int_option_t option) /**< the converting policies */
 {
-  if (isnan (*d))
+  if (*d != *d) /* isnan (*d) */
   {
```

Comment:
Maybe we could have a more descriptive comment referencing to the issue
to avoid changing back to isnan in the future by a greedy contributor.
Otherwise LGTM (informal).

Reply:
Extended the comment.